Training25/Program.cs

```diff
@@ -3,11 +3,44 @@
 // Copyright (c) Metamation India.
 // ------------------------------------------------------------------
 // Program.cs
-// Program on main branch.
+// Program on T11 - Armstrong number.
 // ------------------------------------------------------------------------------------------------
 namespace Training25;
+
 internal class Program {
    static void Main (string[] args) {
-      Console.WriteLine ("Hello, World!");
+      string input = GetInput ();
+      ArmStrong (input);

```

Comment:
`input ` is used once so better it can be :

```suggestion
      ArmStrong (GetInput ());
```